mailbox/store/src/main/java/org/apache/james/mailbox/store/quota/ListeningCurrentQuotaUpdater.java

```diff
@@ -107,7 +107,7 @@ private void handleExpungedEvent(Expunged expunged, QuotaRoot quotaRoot) {
 
     private void handleAddedEvent(Added added, QuotaRoot quotaRoot) {
         computeQuotaOperation(added, quotaRoot).ifPresent(Throwing.<QuotaOperation>consumer(quotaOperation -> {
-            currentQuotaManager.increase(quotaOperation);
+            currentQuotaManager.increase(quotaOperation).block();
```

Comment:
We sould avoid a double block. (eventBus call below)

A.then(B).block will prevent this.